soda/core/soda/common/json_helper.py

```diff
@@ -56,7 +56,7 @@ def to_jsonnable(o):
         if isinstance(o, Decimal):
             return float(o)
         if isinstance(o, datetime.datetime):
-            return o.isoformat(timespec="seconds")
+            return o.strftime("%Y-%m-%dT%H:%M:%S.%fZ")
```

Comment:
Are you sure we need the micro seconds? I thought cloud was only accepting until seconds

Reply:
I think the better option instead of adding Z is to use :

`o.astimezone(timezone.utc).isoformat(timespec="seconds")`

Reply:
The above would return:  `2022-06-14T09:46:17+00:00`

Reply:
👍 i ll change it.